extension/persistence/relational-jdbc/build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+fun isValidDep(dep: String): Boolean {
+  val depRegex = "^[\\w.]+:[\\w\\-.]+:[\\w\\-.]+$".toRegex()
+  return dep.matches(depRegex)
+}
+
+plugins {
+  id("polaris-server")
+  alias(libs.plugins.quarkus)
+  alias(libs.plugins.jandex)
+}
+
+dependencies {
+  implementation(project(":polaris-core"))
+  implementation("org.apache.commons:commons-dbcp2:2.9.0")
+
+  val relationJdbcDeps: String? = project.findProperty("relationalJdbcDeps") as String?
+  relationJdbcDeps?.let {
+    val dependenciesList = it.split(",")
+    dependenciesList.forEach { dep ->
+      val trimmedDep = dep.trim()
+      if (isValidDep(trimmedDep)) {
+        implementation(trimmedDep)
+      } else {
+        throw GradleException("Invalid dependency format: $trimmedDep")
+      }
+    }
+  }
+
+  implementation(platform(libs.quarkus.bom))
+  implementation("io.quarkus:quarkus-core")
```

Comment:
Could you clarify why Quarkus Core needs to be included here? (without code it's not easy to deduce :) )

Reply:
I agree, and apologies for that, need it for CDI's and all other benfits quarkus has.
The very reasons why we introduced in Eclipselink in pr : https://github.com/apache/polaris/pull/469
Idea is to have a class like this which we can invoke via quarkus 

```
@ApplicationScoped
@Identifier("relational-jdbc")
public class JdbcMetaStoreManagerFactory implements MetaStoreManagerFactory {
```

Reply:
Can this dep be removed then?